podcasts/Notifications/NotificationsCoordinator.swift

```diff
@@ -234,18 +234,62 @@ enum NotificationsGroup: CaseIterable {
         }
     }
 
+    func trigger(order: Int, notification: NotificationType) -> UNNotificationTrigger? {
+        if Self.speedUpNotifications {
+            return UNTimeIntervalNotificationTrigger(timeInterval: Double(order + 1) * timeIntervalStep, repeats: notification.isRepeatable)
+        }
+        let todayComponents = Calendar.current.dateComponents(in: .current, from: Date.now)
+        let todayWeekday: Int = todayComponents.weekday ?? 1
```

Comment:
I believe you can avoid the optional here:

```suggestion
        let todayWeekday = Calendar.current.component(.weekday, from: Date.now)
```